doc/man1/flux-jobs.rst

```diff
@@ -187,7 +187,7 @@ the following conversion flags are supported by *flux-jobs*:
 
 **!F**
    convert a duration in floating point seconds to Flux Standard Duration (FSD).
-   string.  Defaults to empty string if duration field does not exist.
```

Comment:
My guess is that this was supposed to say something like "to a Flux Standard Duration (FSD) string" since it is important to know the conversion results in a string, not a number or other object that has special format specifiers. Definitely not a big deal, but it would be nice to keep the emphasis that the result of the conversion is a string.

Reply:
Yeah, also a missing `a` or something. Definitely needed a fix.